duckduckgo/iOS
# Feature/elle/fire button animations

Task/Issue URL: https://app.asana.com/0/72649045549333/1199145319462726/f
Tech Design URL:
CC: @brindy, @bwaresiak 

**Description**:
Changes the fire animation to use the new fire animation, and adds a settings entry to swap between three different animations (and none).

Design review is done, but product review is ongoing

Things to note are:
- This changes the timing of the various fire button actions (e.g. when the keyboard appears, when the toast appears), so we need to make sure we're happy with that.
- We reverted to an older version of Lottie for performance reasons 
- There's a minor change to how TrackerDataManager and ContentBlockerUserScript interact for performance reasons, we need to ensure that hasn't broken anything
- We've had a lot of back and forth on if the performance is good enough, so need to make sure we are happy with it.

**Steps to test this PR**:
1. Test each new fire animation (including none!), from everywhere it's possible to press the fire button, and check it actually clears what it's supposed to. 
2. Pay particular attention to if the performance is good enough
3. Check that the content blocking still works correctly, and still updates correctly
4. Check any other Lottie animations



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
